src/genome_builds.php

```diff
@@ -132,68 +132,37 @@
-                        'active' => in_array('position_g_start_' . $_POST['column_suffix'], $aActiveColumnsVariants),
-                    ),
-                    'position_g_end_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_start_' . $_POST['column_suffix'],
-                        'active' => in_array('position_g_end_' . $_POST['column_suffix'], $aActiveColumnsVariants)
-                    ),
+                    'VariantOnGenome/DNA/' . $_POST['column_suffix'] => 'VARCHAR(255)',
+                    'position_g_start_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_end',
+                    'position_g_end_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_start_' . $_POST['column_suffix'],
                 ),
                 TABLE_TRANSCRIPTS => array(
-                    // TODO: Ask Ivo whether or not the mrna positions should be NOT NULL
-                    'position_g_mrna_start_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_mrna_start',
-                        'active' => in_array('position_g_mrna_start_' . $_POST['column_suffix'], $aActiveColumnsTranscripts)
-                    ),
-                    'position_g_mrna_end_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_mrna_end',
-                        'active' => in_array('position_g_mrna_end_' . $_POST['column_suffix'], $aActiveColumnsTranscripts)
-                    ),
+                    'position_g_mrna_start_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_mrna_end',
+                    'position_g_mrna_end_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_mrna_start_' . $_POST['column_suffix'],
                 ),
             );
 
-            // Add columns to VOG and Transcripts tables.
-            foreach (array_keys($aTablesAndTheirColumns) as $sTable) {
-                $bToAdd = False;
+            // Add the columns to the VOG and Transcripts tables.
+            foreach ($aTablesAndTheirColumns as $sTable => $aTable) {
+                $aActiveColumns = $_DB->query('
+                    SELECT COLUMN_NAME
+                    FROM information_schema.COLUMNS
+                    WHERE TABLE_SCHEMA = DATABASE()
+                      AND TABLE_NAME = ?
+                      AND COLUMN_NAME IN (' . implode(',', array_fill(0, count($aTable), '?')) . ')',
+                    array_merge(array($sTable), array_keys($aTable)))->fetchAllColumn();
+
+                $bToAdd = false;
                 $sSQL = 'ALTER TABLE ' . $sTable;
 
-                foreach (array_keys($aTablesAndTheirColumns[$sTable]) as $sColumn) {
-                    $aColumn = $aTablesAndTheirColumns[$sTable][$sColumn];
-                    if (!$aColumn['active']) {
-                        $bToAdd = True;
```

Comment:
`True` and `False` feel a bit "Python-ish", and can simply be written as `true` and `false`.